scripts/src/logseq/tasks/dev.clj

```diff
@@ -17,3 +20,18 @@
                "bb lint:ns-docstrings"]]
     (println cmd)
     (shell cmd)))
+
+
+(defn gen-malli-kondo-config
```

Comment:
This command failed for me with with exception below.  I don't need to generate these yet but at some point it'd be good to make sure this works for others
<details>
```sh

clojure -M:cljs compile gen-malli-kondo-config
[:gen-malli-kondo-config] Compiling ...
[:gen-malli-kondo-config] Build completed. (173 files, 4 compiled, 2 warnings, 2.91s)

[1m------ WARNING #1 - :undeclared-var --------------------------------------------[0m
 File: /Users/me/code/work/logseq/src/main/gen_malli_kondo_config/core.cljs:10:3
--------------------------------------------------------------------------------
[2m   7 | [0m
[2m   8 | [0m
[2m   9 | (defn main [& _args][0m
[1m  10 |   (collect-schema)[0m
---------^----------------------------------------------------------------------
 [33;1mUse of undeclared Var malli.instrument/collect![0m
--------------------------------------------------------------------------------
[2m  11 |   (println (mc/linter-config (mc/collect-cljs)))[0m
[2m  12 |   (js/process.exit 0))[0m
[2m  13 | [0m
--------------------------------------------------------------------------------

[1m------ WARNING #2 - :undeclared-var --------------------------------------------[0m
 File: /Users/me/code/work/logseq/src/main/gen_malli_kondo_config/core.cljs:10:3
--------------------------------------------------------------------------------
[2m   7 | [0m
[2m   8 | [0m
[2m   9 | (defn main [& _args][0m
[1m  10 |   (collect-schema)[0m
---------^----------------------------------------------------------------------
 [33;1mUse of undeclared Var gen-malli-kondo-config.core/cljs[0m
--------------------------------------------------------------------------------
[2m  11 |   (println (mc/linter-config (mc/collect-cljs)))[0m
[2m  12 |   (js/process.exit 0))[0m
[2m  13 | [0m
--------------------------------------------------------------------------------
generate kondo config:  .clj-kondo/metosin/malli-types/config.edn
This browser doesn't support requestAnimationFrame. Make sure that you load a polyfill in older browsers. https://reactjs.org/link/react-polyfills
This browser doesn't support cancelAnimationFrame. Make sure that you load a polyfill in older browsers. https://reactjs.org/link/react-polyfills
SHADOW import error /Users/me/code/work/logseq/.shadow-cljs/builds/gen-malli-kondo-config/dev/out/cljs-runtime/shadow.module.main.append.js

/Users/me/code/work/logseq/.shadow-cljs/builds/gen-malli-kondo-config/dev/out/cljs-runtime/gen_malli_kondo_config/core.cljs:9
(defn main [& _args]
^
TypeError: Cannot read properties of undefined (reading 'collect_BANG_')
    at Function.gen-malli-kondo-config.core/main [as cljs$core$IFn$_invoke$arity$variadic] (/Users/me/code/work/logseq/.shadow-cljs/builds/gen-malli-kondo-config/dev/out/cljs-runtime/gen_malli_kondo_config/core.cljs:9:1)
    at f (/Users/me/code/work/logseq/.shadow-cljs/builds/gen-malli-kondo-config/dev/out/cljs-runtime/gen_malli_kondo_config/core.cljs:9:1)
    at cljs.core/apply-to (/Users/me/code/work/logseq/.shadow-cljs/builds/gen-malli-kondo-config/dev/out/cljs-runtime/cljs/core.cljs:3917:1)
    at Function.cljs$core$IFn$_invoke$arity$2 (/Users/me/code/work/logseq/.shadow-cljs/builds/gen-malli-kondo-config/dev/out/cljs-runtime/cljs/core.cljs:3969:11)
    at /Users/me/code/work/logseq/.shadow-cljs/builds/gen-malli-kondo-config/dev/out/cljs-runtime/shadow.module.main.append.js:3:17
    at global.SHADOW_IMPORT (/Users/me/code/work/logseq/static/gen-malli-kondo-config.js:64:44)
    at /Users/me/code/work/logseq/static/gen-malli-kondo-config.js:1668:1
    at Object.<anonymous> (/Users/me/code/work/logseq/static/gen-malli-kondo-config.js:1670:3)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
Error while executing task: dev:gen-malli-kondo-config
```
</details>

Reply:
@logseq-cldwalker I can reproduce it after pulling master branch.
and fix here https://github.com/logseq/logseq/pull/8388